packages/common/src/associations/removeAssociation.ts

```diff
@@ -1,22 +0,0 @@
-import { IWithAssociations } from "../core/traits/IWithAssociations";
-import { IAssociationInfo } from "./types";
-
-/**
- * Remove an association from an entity
- * @param info
- * @param entity
- * @returns
- */
-export function removeAssociation(
```

Comment:
Similarly, I can deprecate this rather than removing it and have it call `breakAssociation`.